### PR TITLE
RR-50 - Implement Logs

### PR DESCRIPTION
**Pull request by nenedac**

### CODE REVIEW

File: src/app/api/_utils/database.ts [hunk 1]
```diff
@@ -89,18 +89,21 @@ export class PostgresClient {
...
     const columns = Object.keys(rows[0]);
 
     return { columns, rows };
   }
 
-  public async insertLog(log: Log): Promise<void> {
+  public async insertLog(organizationName: string,
+    rotationName: string, log: Log): Promise<void> {
+
+    console.info(`Querying logs from: ${organizationName}, ${rotationName}`);
```
Comment (marandino) [inline src/app/api/_utils/database.ts:103]:
```suggestion
    console.info(`Writing alog for: ${organizationName}, ${rotationName}`);
```

File: src/app/api/_utils/logic.ts [hunk 2]
```diff
@@ -2,13 +2,13 @@ import { Log, LogType, SlackUser } from 'types';
 
 export function createLog(
   description: string,
-  executedBy: string,
+  executed_by: string,
   type: LogType
 ): Log {
   return {
     description,
-    date: Date.now(), // Automatically sets the current timestamp in milliseconds
```
Comment (marandino) [inline src/app/api/_utils/logic.ts:10]: leave the comment, it's not bad

File: src/app/api/v1/slack/rotation-post/route.ts [hunk 3]
```diff
@@ -8,43 +8,56 @@ import {
   parsePayloadFromRequest,
   sanitizeSlackText,
 } from 'utils/slack';
+import { createLog } from '@/app/api/_utils/logic';
```
Comment (marandino) [inline src/app/api/v1/slack/rotation-post/route.ts:11]:
```suggestion
import { createLog } from 'utils/logic';
```

**Review by marandino — APPROVED** [review]
lgtm, please test it locally before merging though

**Review by K3iiX — APPROVED** [review]
YLVB

**Pull request merged**